turbine/src/broadcast_stage/broadcast_duplicates_run.rs

```diff
@@ -169,18 +177,25 @@ impl BroadcastRun for BroadcastDuplicatesRun {
         )
         .expect("Expected to create a new shredder");
 
+        // Chained Merkle shreds are always discarded in epoch 0, due to
+        // feature_set::enable_chained_merkle_shreds. Below can be removed once
+        // the feature gated code is removed.
+        let should_chain_merkle_shreds = bank.epoch() > 0;
```

Comment:
to confirm: this was the change that now allows the local cluster tests to pass? 
I took a look carl's concern here https://github.com/solana-labs/solana/pull/35115#discussion_r1480783646 and it should not be a problem for the test.

Reply:
yes, that is correct.